pennylane_forest/converter.py

```diff
@@ -327,7 +327,9 @@ class ProgramLoader:
     _matrix_dictionary = pyquil.simulation.matrices.QUANTUM_GATES
 
     def __init__(self, program):
-        self.program = program
+        raise ValueError("The PyQuil program conversion feature is currently not supported.")
```

Comment:
:+1:

It might be worth removing the device card on the QPU from the docs! and maybe also removing the converter.

Another option could be to simply have a `.. warning::` box that the converter is not support with the latest version of PL

Reply:
Could we raise only if version 0.14 or above of PL?

Reply:
Updated to do that 👍